### DISCUSSION TITLE
Publish release note directly on GitHub

### DISCUSSION THREAD

**Discussion by MartinDelille** [Kong/insomnia#3773] [Feedback]
**Is your feature request related to a problem? Please describe.**

I watch only the release on GitHub. The current release note contains only a link to the website. 

Why don’t you post a release note directly where GitHub users expect it?

It would remove one click for getting the information!

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by gschier** [comment]
Not saying this as a "no" but the current reason is that, because of the way the changelog is managed, it's non-trivial to automate posting it to GitHub. Since most users don't even know the app is open source, it made sense to prioritize the website over GitHub. In summary, I'm lazy.

We'll likely be change the way changelogs are managed in the future so I'll definitely keep this in mind. Ideally they would be in both places 👍

**Comment by MartinDelille** [comment]
Good developers are all lazy!

I wanted to contribute to your project by making this suggestion. I'm wondering if there is a good practice concerning release notes automation because I see more and more people listing commit or pull request describing the changes done since last release.

You've really done an amazing work on Insomnia by the way!

**Comment by nijikokun** [comment]
Could have the site (https://github.com/kong/insomnia.rest) source from the releases release notes; wouldn't be too different from the npm package sourcing
